script/RumpelConfig.sol

```diff
@@ -630,6 +657,248 @@ library RumpelConfig {
+        configs[11].selectorStates[0] =
+            SelectorState({selector: IMellow.deposit.selector, state: RumpelGuard.AllowListState.ON});
+        configs[11].selectorStates[1] =
+            SelectorState({selector: IMellow.registerWithdrawal.selector, state: RumpelGuard.AllowListState.ON});
+
+        // Mellow hcETH
+        configs[12] = ProtocolGuardConfig({target: MAINNET_MELLOW_ISETH, selectorStates: new SelectorState[](2)});
+        configs[12].selectorStates[0] =
+            SelectorState({selector: IMellow.deposit.selector, state: RumpelGuard.AllowListState.ON});
+        configs[12].selectorStates[1] =
+            SelectorState({selector: IMellow.registerWithdrawal.selector, state: RumpelGuard.AllowListState.ON});
+
+        // Mellow hcETH
+        configs[13] = ProtocolGuardConfig({target: MAINNET_MELOW_SIBTC, selectorStates: new SelectorState[](2)});
+        configs[13].selectorStates[0] =
+            SelectorState({selector: IMellow.deposit.selector, state: RumpelGuard.AllowListState.ON});
+        configs[13].selectorStates[1] =
+            SelectorState({selector: IMellow.registerWithdrawal.selector, state: RumpelGuard.AllowListState.ON});
+
+        // Mellow hcETH
+        configs[14] = ProtocolGuardConfig({target: MAINNET_MELLOW_LUGAETH, selectorStates: new SelectorState[](2)});
+        configs[14].selectorStates[0] =
+            SelectorState({selector: IMellow.deposit.selector, state: RumpelGuard.AllowListState.ON});
+        configs[14].selectorStates[1] =
+            SelectorState({selector: IMellow.registerWithdrawal.selector, state: RumpelGuard.AllowListState.ON});
+
+        // Mellow hcETH
+        configs[15] = ProtocolGuardConfig({target: MAINNET_MELLOW_ROETH, selectorStates: new SelectorState[](2)});
+        configs[15].selectorStates[0] =
+            SelectorState({selector: IMellow.deposit.selector, state: RumpelGuard.AllowListState.ON});
+        configs[15].selectorStates[1] =
+            SelectorState({selector: IMellow.registerWithdrawal.selector, state: RumpelGuard.AllowListState.ON});
+
+        // Mellow hcETH
+        configs[16] = ProtocolGuardConfig({target: MAINNET_MELLOW_RSUNIBTC, selectorStates: new SelectorState[](2)});
+        configs[16].selectorStates[0] =
+            SelectorState({selector: IMellow.deposit.selector, state: RumpelGuard.AllowListState.ON});
+        configs[16].selectorStates[1] =
+            SelectorState({selector: IMellow.registerWithdrawal.selector, state: RumpelGuard.AllowListState.ON});
+
+        return configs;
+    }
+
+    function getMellowVaultsGuardTokenConfigs() internal pure returns (TokenGuardConfig[] memory) {
+        TokenGuardConfig[] memory configs = new TokenGuardConfig[](17);
+
+        configs[0] = TokenGuardConfig({
+            token: MAINNET_MELLOW_DVSTETH,
+            transferAllowState: RumpelGuard.AllowListState.ON,
+            approveAllowState: RumpelGuard.AllowListState.ON
```

Comment:
any reason this one adds approve while the others don't?

Reply:
Fixed